server/views/applications/pages/risk-to-self/historical-risk.njk

```diff
@@ -2,6 +2,11 @@
 {% set pageName = "historical-risk" %}
 
 {% block questions %}
+  {% if page.importDate %}
+    <p>Imported from OASys on <strong>{{page.importDate}}</strong>
```

Comment:
it looks like the prototype design has this in grey text - should we add that styling here? I'm assuming it's a GDS approved grey so should meet accessibility standards...

Reply:
good spot 🕵️